apps/dashboard/src/app/resources/resource-list.component.ts

```diff
@@ -325,4 +339,46 @@ export class ResourceListComponent implements OnInit, OnDestroy {
       });
     }
   }
+
+  updateLabelList($event: LabelValue[]) {
+    this.currentLabelList = $event;
+  }
+
+  addLabelsToSelection() {
```

Comment:
I found 2 things, but don't know if they're important:
- The resource modification request is done even if the user doesn't select any label in the dropdown.
- After adding labels to a resource, the "classification" column doesn't update.

Reply:
> The resource modification request is done even if the user doesn't select any label in the dropdown.

good point, I'll update the code to prevent that

> After adding labels to a resource, the "classification" column doesn't update. 

The update is triggered but there is a bug in the backend with x-synchronous which makes us doing the refresh too soon. Once fixed on the backend, the front should work as well.